frontend/src/redux/reducers/ui.tsx

```diff
@@ -163,8 +163,18 @@ function reducer(state = INITIAL_STATE, action: Action) {
     }
     case UI_SET_DETAILS_VIEW: {
       const { sectionName, action: sectionFunc } = action;
-      const detailViews = [...newFilters.views.details.pluginAppendedDetailViews];
-      detailViews.push({ sectionName, sectionFunc });
+      const pluginDetailViews = newFilters.views.details.pluginAppendedDetailViews;
+      const detailViews = [...pluginDetailViews];
+      // only push if it's a unique section
+      if (
+        detailViews.length === 0 ||
+        !detailViews.find(
+          (view: { sectionName: string; sectionFunc: sectionFunc }) =>
+            view.sectionName === sectionName
+        )
+      ) {
+        detailViews.push({ sectionName, sectionFunc });
```

Comment:
I think this cures the symptom, not the cause, of somehow getting the register* functions called more than once for existing plugins. Maybe it's worth investigating further to realize exactly why this is happening.

If this solution should still be added, then I think the correct thing to do may be to replace the entry in the details views, so we can replace any older code with new one.